src/adapters/bidderFactory.js

```diff
@@ -286,12 +286,14 @@ export function newBidder(spec) {
   function isInvalidAlternateBidder(responseBidder, requestBidder) {
     let allowAlternateBidderCodes = bidderSettings.get(requestBidder, 'allowAlternateBidderCodes') || false;
     let alternateBiddersList = bidderSettings.get(requestBidder, 'allowedAlternateBidderCodes');
+    let adjustForAlternateBids = bidderSettings.get(requestBidder, 'adjustForAlternateBids') || false;
     if (!!responseBidder && !!requestBidder && requestBidder !== responseBidder) {
       alternateBiddersList = isArray(alternateBiddersList) ? alternateBiddersList.map(val => val.trim().toLowerCase()).filter(val => !!val).filter(uniques) : alternateBiddersList;
-      if (!allowAlternateBidderCodes || (isArray(alternateBiddersList) && (alternateBiddersList[0] !== '*' && !alternateBiddersList.includes(responseBidder)))) {
+      if ((!adjustForAlternateBids && !allowAlternateBidderCodes) || !allowAlternateBidderCodes || (isArray(alternateBiddersList) && (alternateBiddersList[0] !== '*' && !alternateBiddersList.includes(responseBidder)))) {
```

Comment:
what's this for - why would `isInvalidAlternateBidder` care about adjustments?

Reply:
See what you mean, reverted my change for this line.